paasta_tools/kubernetes_tools.py

```diff
@@ -815,25 +822,23 @@ def get_kubernetes_services_running_here_for_nerve(
     nerve_list = []
     for kubernetes_service in kubernetes_services:
         try:
-            kubernetes_service_config = load_kubernetes_service_config(
-                service=kubernetes_service.name,
-                instance=kubernetes_service.instance,
-                cluster=cluster,
-                load_deployments=False,
-                soa_dir=soa_dir,
-            )
-            for registration in kubernetes_service_config.get_registrations():
+            for registration in kubernetes_service.registrations:
                 reg_service, reg_namespace, _, __ = decompose_job_id(registration)
-                nerve_dict = load_service_namespace_config(
-                    service=reg_service, namespace=reg_namespace, soa_dir=soa_dir,
-                )
+                try:
+                    nerve_dict = load_service_namespace_config(
+                        service=reg_service, namespace=reg_namespace, soa_dir=soa_dir,
+                    )
+                except Exception:
+                    log.warning(f"Could not get smartstack config for {reg_service}.{reg_namespace}, skipping")
```

Comment:
Please print the exception info in the warning, so we won't wonder what exactly happened here.